docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+COPY console.sh /bin/console
+RUN chmod 755 /bin/console
+ADD https://keybase.io/circonuspkg/pgp_keys.asc?fingerprint=14ff6826503494d85e62d2f22dd15eba6d4fa648 /tmp/circonus-apt.key
+RUN apt-key add /tmp/circonus-apt.key
+RUN rm -f /tmp/circonus-apt.key
+COPY circonus.list /etc/apt/sources.list.d/circonus.list
+RUN apt-get update
+RUN apt-get install -y circonus-field-broker
+RUN apt-get install -y --allow-unauthenticated circonus-field-broker-crashreporter
+RUN apt-get install -y telnet
```

Comment:
These should be combined into a single RUN ala:

```
RUN apt-get update && \
apt-get install -y circonus-field-broker && \
apt-get install -y --allow-unauthenticated circonus-field-broker-crashreporter && \
apt-get install -y telnet && \
&& rm -rf /var/lib/apt/lists/*
```

This keeps the image size small instead of adding a new layer for each `RUN` command.

Reply:
Further consolidated them.